frontend/www/js/omegaup/components/user/Profile.vue

```diff
@@ -135,6 +163,31 @@ export default class UserProfile extends Vue {
     return this.groupElements(this.unsolvedProblems, this.columns);
   }
 
+  get badges(): omegaup.Badge[] {
+    const badges: omegaup.Badge[] = [];
+    const unlockedBadges: omegaup.Badge[] = Array.from(this.profileBadges)
+      .filter(x => this.visitorBadges.has(x))
+      .map(x => {
+        return { badge_alias: x, unlocked: true };
+      });
+    const lockedBadges: omegaup.Badge[] = Array.from(this.profileBadges)
+      .filter(x => !this.visitorBadges.has(x))
+      .map(x => {
+        return { badge_alias: x, unlocked: false };
+      });
+    return unlockedBadges
+      .concat(lockedBadges)
+      .sort((a: omegaup.Badge, b: omegaup.Badge) => {
+        if (a.badge_alias > b.badge_alias) {
```

Comment:
aquí puedes hacerlo un poco más conciso:

```suggestion
        if (a.badge_alias == b.badge_alias) {
          return 0;
        }
        return (a.badge_alias < b.badge_alias) ? -1 : 1;
```